index.ts

```diff
@@ -14,7 +15,10 @@ open.then((conn) => conn.createChannel()).then((ch) =>
         ch.ack(msg)
         try {
           const message = JSON.parse(msg.content.toString())
-          runInNewContext(message.code, { console: Object.freeze({ ...console }) }, { filename: 'sandboxed.js', timeout: 1e5 })
+          runInNewContext(message.code, {
+            console: deepFreeze({ ...require('console') }),
+            http: deepFreeze({ ...require('http') }),
```

Comment:
you can use `import` on top of the file.. if you want you can move the context object in a dedicated file.

Reply:
Nope, `deepFreeze(http)` affects the existed modules that are used at `amqp` so we can't `import` the modules in the top context. I will rewrite this with `cluster` that allows node using a different scope.